src/components/user-panel/TheUserPanel.vue

```diff
@@ -87,24 +89,41 @@ import Vue from 'vue'
 import Component from 'vue-class-component'
 import UserList from './UserList.vue'
 import { namespace } from 'vuex-class'
-import { User } from '@/store/types'
-import { SocketUserGetters } from '@/store/modules/socket/user'
+import { User, RoleTypes } from '@/store/types'
+import { SocketUserGetters, SocketUserAction } from '@/store/modules/socket/user'
 import { Namespaces } from '@/store'
 
 interface UserTab {
   title: string;
   filter: 'online' | 'offline';
 }
 
-const socketUser = namespace(Namespaces.SOCKET_USER)
+const SocketUser = namespace(Namespaces.SOCKET_USER)
 
 @Component({
   name: 'TheUserPanel',
   components: { UserList }
 })
 export default class TheUserPanel extends Vue {
-  @socketUser.Getter(SocketUserGetters.ONLINE_USERS) onlineUsers!: User[]
-  @socketUser.Getter(SocketUserGetters.OFFLINE_USERS) offlineUsers!: User[]
+  @SocketUser.Getter(SocketUserGetters.ONLINE_USERS) onlineUsers!: User[]
+  @SocketUser.Getter(SocketUserGetters.OFFLINE_USERS) offlineUsers!: User[]
+  @SocketUser.Getter(SocketUserGetters.IS_AUTHORISED) isAuthorised!: boolean
+  @SocketUser.Action(SocketUserAction.SET_USER) setUser!: (user: User) => void
+
+  created () {
+    this.setUser({
```

Comment:
This is to allow you to create Tactics and look at the User list. It is hardcoded until we have the Socket Room State implemented.

Please edit the JTI and put your user's JTI.